scripts/weekly-tileset-generation.py

```diff
@@ -0,0 +1,60 @@
+import subprocess
+
+import hyp3_sdk as sdk
+import requests
+from get_frame_list_from_cmr import get_frames_for_direction
+
+
+def publish_mosaic(job: sdk.Job) -> None:
+    source = f's3://hyp3-opera-disp-sandbox-contentbucket-ibxz8lcpdo59/{job.job_id}/tms/'
+    target = f's3://asf-services-web-content-prod/main/{job.name}/'
+    subprocess.run(['aws', '--profile', 'edc-prod', 's3', 'sync', source, target], check=True)
+
+
+def build_job(name: str, measurement_type: str, frames: list[int]) -> dict:
+    return {
+        'job_type': 'OPERA_DISP_TMS',
+        'name': name,
+        'job_parameters': {
+            'measurement_type': measurement_type,
+            'frame_ids': frames,
+            'start_date': '20140101',
+            'end_date': '20260101',
+        },
+    }
+
+
+def main():
+    ascending_frames = get_frames_for_direction('ASCENDING')
+    descending_frames = get_frames_for_direction('DESCENDING')
+
+    # frame 21518 has disjoint temporal coverage
+    if 21518 in ascending_frames:
+        ascending_frames.remove(21518)
+
+    HyP3 = sdk.HyP3('https://hyp3-opera-disp-sandbox.asf.alaska.edu/')
+
+    prepared_jobs = [
+        build_job('disp/desc', 'displacement', descending_frames),
+        build_job('disp/asc', 'displacement', ascending_frames),
+        build_job('vel/desc', 'velocity', descending_frames),
+        build_job('vel/asc', 'velocity', ascending_frames),
```

Comment:
@jacquelynsmale looks like we got the prefix order backwards here, should have been `desc/disp` instead of `disp/desc`. The displacement app is looking for tiles under URLs like https://d3g9emy65n853h.cloudfront.net/main/asc/disp/3/2/4.png. Can you push a fix for that when you have time today?